src/data/move-attrs/multi-hit-attr.ts

```diff
@@ -89,10 +88,7 @@ export class MultiHitAttr extends MoveAttr {
         const party = user.getParty();
         // No status means the ally pokemon can contribute to Beat Up
         return party.reduce((total, pokemon) => {
-          return (
-            total
-            + (pokemon.id === user.id ? 1 : pokemon?.status && pokemon.status.effect !== StatusEffect.NONE ? 0 : 1)
-          );
+          return total + (pokemon.getStatusEffect(true) ? 0 : 1);
```

Comment:
that doesn't seem strictly equal?
Before it would include the  user even if it is statuses, but with this change it wouldn't, which based on bulba doesn't seems right
> from the user and from each conscious Pokémon in the user's [party](https://bulbapedia.bulbagarden.net/wiki/Party) that does not have a [non-volatile status](https://bulbapedia.bulbagarden.net/wiki/Status_condition#Non-volatile_status)

```suggestion
          return total + (pokemon.id === user.id ? 1 : (pokemon.getStatusEffect(true) ? 0 : 1));
```